test/parallel/taskCompare/elliot/empty-omp-parallel-for.skipif

```diff
@@ -0,0 +1,6 @@
+# skip configs where -fopenmp or /proc/cpuinfo aren't supported
+CHPL_TARGET_COMPILER==clang
+CHPL_TARGET_COMPILER<=pgi
```

Comment:
I think there might be other configs that may need to be skipped, e.g. I think icc's openmp option is just '-openmp'.  I'm not sure I can accurately name what all needs to be skipped though, and would be tempted to let it run for a night to find out.